partner_default_sale_discount/models/sale_order.py

```diff
@@ -31,12 +31,8 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
         xml_discount = eview.xpath("//field[@name='default_sale_discount']")
         if xml_order_line and xml_discount:
             # This should be handled in "string" mode
```

Comment:
If it were enough, I wouldn't have asked before :wink: 

If you don't explain why it should be handled as a string, then a future code reviewer can believe this is overengineered. Simply say `This should be handled in "string" mode to avoid conflicts in case more than 1 addon modify the field's or view's context` and all will be clear.